l10n_es_vat_book/report/vat_book_invoices_issued.xml

```diff
@@ -27,12 +27,12 @@
               <t t-call="l10n_es_vat_book.vat_book_contact"/>
             </div>
             <div class="row">
-              <div class="col-xs-12 mt32" id="title_invoices_issued">
+              <div class="col-12 mt32" id="title_invoices_issued">
                 <h4>Issued Invoices</h4>
               </div>
             </div>
             <div class="row">
-              <div class="col-xs-12" id="invoices_issued_detail_div">
+              <div class="col-12" id="invoices_issued_detail_div">
                 <table class="table table-condensed"
```

Comment:
You must change table-condensed by table-sm

Reply:
Done